frame/bounties/src/lib.rs

```diff
@@ -440,15 +504,23 @@ decl_module! {
 								}
 							},
 						}
+
+						// Unreserve the fee for Curator which got reserved during "accept_curator()"
+						let bounty_account = Self::bounty_account_id(bounty_id);
+						T::Currency::unreserve(&bounty_account, bounty.fee);
+
 					},
 					BountyStatus::PendingPayout { ref curator, .. } => {
 						// The bounty is pending payout, so only council can unassign a curator.
 						// By doing so, they are claiming the curator is acting maliciously, so
 						// we slash the curator.
 						ensure!(maybe_sender.is_none(), BadOrigin);
 						slash_curator(curator, &mut bounty.curator_deposit);
+						// Unreserve the fee for Curator which got reserved during "accept_curator()"
+						let bounty_account = Self::bounty_account_id(bounty_id);
```

Comment:
what is this?

Reply:
In this context Sequence no 60, 61 ("unassign_curator()"), 
curator fee gets unreserved, which gets reserved during "accept_curator()"
sequence no 4, 5. **When bounty state is in "PendingPayout"**

![](https://i.imgur.com/nQH8cEM.png)

![](https://i.imgur.com/y9xwDxh.png)

Reply:
As suggested, design for subbounty fund management got changed, Now each subbounty got unique account.
Code is removed.